### ISSUE TITLE
quick install: Rekall install with pip fails on --pre on 12.04?

### ISSUE THREAD

**Issue by adricnet** [google/grr#36]
Seems the pip in Ubuntu 12.04 doesn't have the install --pre option, though others do, such as pip-2.7 on my Mac.

A quick edit to the install script got rekall installed and the quick install moving along. It's line 127 in the install_script_ubuntu.sh I wget'd in a couple hours ago as per the quick install [adoc](https://github.com/google/grr-doc/blob/master/quickstart.adoc).

```
header "Installing Rekall"
run_cmd_confirm sudo pip install --upgrade rekall #--pre
```

If you want a pull req (or patch) for this please tell me where to send it.

Thanks,
adric

**Comment by brifordwylie** [comment]
Just an unsolicited comment on this, I noticed that the pip that comes with python 2.7.9 does not seem to recognize the --pre argument (btw Travis CI is now using 2.7.9). So there's literally no way to pip install distorm3 (with or without --pre arg). So just for fun I looked into it. distorm3 setup.py has version : '3' in it. This appears to make pypi think that it's a pre-release. So simply changing that to version: '3.0' does the trick (pip install distorm3-test to try it out). I've sent an email with this info to  Gil Dabah.
